e2e/wsl-integrations.e2e.spec.ts

```diff
@@ -15,6 +15,7 @@ import { createDefaultSettings, packageLogs, reportAsset } from './utils/TestUti
 import { spawnFile } from '@/utils/childProcess';
 
 test.describe('WSL Integrations', () => {
+  test.skip(true, 'TODO');
```

Comment:
I guess this kinda serves the same purpose as a comment with a TODO in it. Do we have an issue that we reference here so we can understand what needs to change when we come back later?

Reply:
Filed #2881 and added that to the comment in c36b69c5cb587d972ba5f43f240a3caca5799b54.